### DISCUSSION TITLE
Posterior Check Facet_by issue

### DISCUSSION THREAD

**Discussion by MZelko82** [lindeloev/mcp#124] [Q&A] [answered]
Given my relative inexperience, not sure if I should put this as an issue.

Both my simulated and real data have 3 columns "id", "x" and "y", all numeric types. When I run pp_check(simfit, facet_by ="id"), it returns the density plots per id; however, when I do the same for the real data, it returns the error(with traceback):

> Error: Only know how to add ggplots and/or grobs
> 3. stop("Only know how to add ggplots and/or grobs", call. = FALSE)
> 2. patchwork::wrap_plots(all_plots)
> 1.mcp::pp_check(fitm5, facet_by = "id")

I tried to debug it myself but haven't had any success so far. Notably, pp_check(realdata) returns the group level density plot.

**Comment by lindeloev** [comment, marked as answer]
Ah yes, it currently only allows for faceting by categorical variables. I'll consider adding support for faceting by numerical ones, especially with the upcoming version which supports multiple regression.

**Reply by MZelko82** [reply]
Ah right, thanks mate. Will try to work around it for now
